### PR TITLE
feat: make the action work

### PR DESCRIPTION
**Pull request by Rignchen**

### CODE REVIEW

**Comment by Rignchen** [comment]
apparently the ``/github/workspace`` isn't mounted in the docker as I thaught I read on the github wiki

```
$ echo "Test: $(diff -rq test/test test/expected)"
diff: can't stat 'test/test': No such file or directory
Test: 

$ tree .
.
0 directories, 0 files

$ pwd
/github/workspace
```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -44,11 +44,33 @@ jobs:
           push: true
           tags: ${{ env.TEST_TAG }}
 
+      - name: Pull tests
```
Comment (fuzoh) [inline .github/workflows/ci.yml:47]: You need to checkout in a subfolder ? Here you overwrite the first checkout repo ?

Comment (Rignchen) [reply .github/workflows/ci.yml:47]: I don't think that's the problem, currently I have these commands to be able to know what command is the bash script executing
```bash
echo "INPUT_LANG: $INPUT_LANG"
echo "sw $INPUT_PATH -e $EXTENSIONS"
```
and when I try to run the docker localy I get
```
INPUT_LANG: rs js
sw test/test -e rs -e js
```
but when it runs in the github action I get
```
INPUT_LANG: $INPUT_LANG
sw $INPUT_PATH -e $INPUT_LANG
```

**Comment by Rignchen** [comment]
the action ``Continuous Integration / Test Container Image (pull_request)`` is finally working! :D
now just need to see why is that linter not happy

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Rignchen) [reply .github/workflows/ci.yml:47]: it's not a problem as the container is built before overwriting the first checkout repo and building the docker is the only thing we need to do from the first checkout

**Review by fuzoh — APPROVED** [review]

**Pull request merged**